float64[] gripperPos
float64[] jointVel
float64 samplingTime